### PR TITLE
Fix top content on chat live  , closes #337

### LINKED ISSUE #337: Same user information
When I open the chat for http://localhost:4000/wintermeyer/messages/Musterfrau I see this on top of that page:

Bildschirmfoto 2024-04-24 um 08 12 34

That doesn't help me as a user. The information from the profile page would give me a bit more background. So please refactor the following part and not only put it in the profile page but also use that as a replacement for the above blue thing for the chat page.

Bildschirmfoto 2024-04-24 um 08 12 16

PS: Please make sure that the heart is clickable and works on all pages it is used.

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
On the chat live , we have information about the receiver and a button to like and unlike

https://github.com/animina-dating/animina/assets/86654131/15746e41-5211-48c4-850a-30c68beb351b

### CODE REVIEW

File: lib/animina_web/components/chat/chat_components.ex [hunk 1]
```diff
@@ -33,10 +34,19 @@ defmodule AniminaWeb.ChatComponents do
...
     <div class="h-[80vh]  w-[100%] ">
       <div class="w-[100%] h-[100%] flex flex-col justify-between">
-        <div class="h-[7%] w-[100%]">
-          <.receiver_profile_box receiver={@receiver} />
+        <div class="h-[10%] w-[100%] z-50">
+          <ProfileComponents.profile_details
+            user={@receiver}
+            current_user={@sender}
+            current_user_has_liked_profile?={@current_user_has_liked_profile?}
+            profile_points={@profile_points}
```
Comment (wintermeyer) [inline lib/animina_web/components/chat/chat_components.ex:42]: This is over bloated. It should be something like:

`<.ProfileComponents.profile_details user={@receiver} current_user={@sender} />`

We digg our own grave with the current approach. The user resource should auto load the flags, the points, etc. No need to have a million arguments for this which could all be part of `user`.

**Review by wintermeyer — CHANGES_REQUESTED** [review]

File: lib/animina_web/components/chat/chat_components.ex (hunk 1 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/components/chat/chat_components.ex:42]: Okay , let me take time to refactor the same thing , same for the profile_live , that should also close #262

Comment (wintermeyer) [reply lib/animina_web/components/chat/chat_components.ex:42]: Do you want me to merge the PR now or do you want to refactor first?

Comment (MICHAELMUNAVU83) [reply lib/animina_web/components/chat/chat_components.ex:42]: I feel the refactor would be a huge task that would take time and focus , best for morning hours , can we leave this open , I do the other small issues to reduce the  issues then focus on this tommorow morning?

Comment (wintermeyer) [reply lib/animina_web/components/chat/chat_components.ex:42]: Let's merge it now and open a new one tomorrow. Done is better than perfect.

**Comment by wintermeyer** [comment]
@MICHAELMUNAVU83 Please create a new issue in case you want one for the refactoring.

**Comment by MICHAELMUNAVU83** [comment]
I have added  #262 to  the MVP milestone

**Pull request merged**